include/Core/UI/Setting/EmailLogSetting.php

```diff
@@ -0,0 +1,121 @@
+<?php
+namespace EmailLog\Core\UI\Setting;
+use EmailLog\Core\UI\Setting\SettingField;
+
+class EmailLogSetting extends Setting {
+
+	/**
+	 * Implement `initialize()` method.
+	 */
+	protected function initialize() {
+		$this->section->id          = 'email-log';
+		$this->section->title       = __( 'Email Log Settings', 'email-log' );
+		$this->section->option_name = 'el_email_log';
+
+		$this->load();
+	}
+
+	/**
+	 * Implement `get_fields()` method.
+	 *
+	 * @return array
+	 */
+	public function get_fields() {
+		$fields = array();
+
+		$email_log_fields = array(
+			'allowed_user_roles' => __( 'Allowed User Roles', 'email-log' ),
+			'remove_email_logs'  => __( 'Remove Data on Uninstall?', 'email-log' ),
```

Comment:
Let's rename this to `remove_on_uninstall`

Reply:
Fixed